lib/recipes/import/food_and_wine.rb

```diff
@@ -15,54 +15,32 @@ def recipe_total_time
         recipe_details['total_time']
       end
 
-      def recipe_ingredients # rubocop:disable Metrics
+      def recipe_ingredients
         document
-          .css('div.mntl-structured-ingredients')
```

Comment:
I think you only needed to change the class name here to `div.mm-recipes-structured-ingredients`. The way it was implemented before was meant to capture multiple lists with headings

For example:
<img width="506" alt="image" src="https://github.com/user-attachments/assets/794bda95-e6d3-4755-89b5-554d030ebf35" />

The new changes aren't really handling the ingredients very well at all

<img width="506" alt="image" src="https://github.com/user-attachments/assets/a1a651d9-a93c-4d7b-beb1-ea79ca202086" />

Some recipes I used to to test
- https://www.foodandwine.com/recipes/rib-eye-aguachile
- https://www.foodandwine.com/recipes/spice-roasted-pork-tenderloin
- https://www.foodandwine.com/lemony-shrimp-dumplings-6422735